rundeckapp/grails-spa/packages/ui-trellis/src/app/components/common/ScheduledExecutionDetails.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div v-if="allowHtml && !firstLineOnly">
+    <div v-if="remainingLine.trim()" data-testid="details-container">
+      <template v-if="mode === 'collapsed' || mode === 'expanded'">
+        <details class="more-info" :open="mode === 'expanded'">
+          <summary>
+            <span :class="textCss">
+              {{ firstLine }}
+            </span>
+            <span
+              class="btn-link btn-xs more-indicator-verbiage"
+              data-testid="see-more-button"
+            >
+              {{ moreText.length ? moreText : $t("more") }}
+            </span>
+            <span
+              class="btn-link btn-xs less-indicator-verbiage"
+              data-testid="see-less-button"
+            >
+              {{ $t("less") }}
+            </span>
+          </summary>
+          <div
+            class="more-info-content"
+            :class="markdownCss"
+            data-testid="more-content"
+          >
+            <VMarkdownView mode="" :content="remainingLine" />
+          </div>
+        </details>
+      </template>
+      <span v-else-if="mode !== 'hidden'" :class="markdownCss">
+        <VMarkdownView mode="" :content="remainingLine" />
+      </span>
+    </div>
+    <span v-else :class="textCss" data-testid="first-line-html-allowed">
+      {{ firstLine }}
+    </span>
+  </div>
+  <span v-else :class="textCss" data-testid="first-line-only">
+    {{ firstLine }}
+  </span>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { VMarkdownView } from "vue3-markdown";
+
+export default defineComponent({
+  name: "ScheduledExecutionDetails",
+  components: {
+    VMarkdownView,
+  },
+  props: {
+    allowHtml: {
+      type: Boolean,
+      default: false,
+    },
+    firstLineOnly: {
+      type: Boolean,
+      default: false,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    mode: {
+      type: String,
+      required: true,
+    },
+    markdownCss: {
+      type: String,
+      default: "",
+    },
+    textCss: {
+      type: String,
+      default: "",
+    },
+    cutoffMarker: {
+      type: String,
+      default: "",
+    },
+    service: {
+      type: String,
+      default: "",
+    },
+    name: {
+      type: String,
+      default: "",
+    },
+    moreText: {
+      type: String,
+      default: "",
+    },
+  },
+  computed: {
+    pluginBaseUrl() {
+      if (this.service.length && this.name.length) {
+        return `/plugin/pluginFile?service=${this.service}&name=${this.name}`;
+      }
+      return "";
+    },
+    replTokens() {
+      // not relevant at the moment
+      return this.pluginBaseUrl.length > 0
+        ? { "plugin.url": this.pluginBaseUrl }
+        : null;
+    },
```

Comment:
is this used? service/name also doesn't seem to make sense in this component.

Reply:
This component would be the equivalent of `scheduledExecution/_description.gsp` and the the replToken would be used in this line:

`<g:markdown><g:autoLink jobLinkId="${jobLinkId}" tokens="${replTokens}">${remainingLine}</g:autoLink></g:markdown>`

It's not relevant for editing/creating a job, but by searching for the template, I did see some places passing the service and name to it, so ended up adding the computed properties, but didn't finish the logic as I was unsure how it would look like with the VMarkdownView component

Reply:
Can remove it for now if you'd like

Reply:
ok, I'd say remove it for now since it is not used in this PR